neqo-crypto/src/result.rs

```diff
@@ -83,7 +83,7 @@ mod tests {
                 assert_eq!(code, -12273);
                 assert_eq!(
                     desc,
-                    "SSL received a record with an incorrect Message Authentication Code."
+                    "SSL received a record with an incorrect Message Authentication Code"
```

Comment:
For my understanding, is our convention to not use a period at the end of panic messages?

I am fine either way.

Reply:
Not sure if we have a stated convention, but I think that's what it looks like based on the code.